dfm/cleaning/deduper.py

```diff
@@ -64,50 +76,59 @@ class Deduper:
 
     def __init__(
         self,
-        split_method: Optional[str] = "char_ngram",
+        split_method: Optional[str] = "word_ngram",
```

Comment:
Ændring af default her kommer nok til at ødelægge tests?

Reply:
Yep, planning to change the tests to fit this change 👍